Added Palo Alto Networks auth config section

vsathiamoo opened the pull request:
Fixed issues with Azure configuration.
Added Palo Alto Networks firewall configuration section details for completeness

azurecla commented:
Hi __@trinadhk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

vsathiamoo committed:
Added Palo Alto Networks auth config section

Fixed issues with Azure configuration.
Added Palo Alto Networks firewall configuration section details for completeness

vsathiamoo committed:
Reply URL Fixed
